src/options/views/edit/index.vue

```diff
@@ -163,6 +192,9 @@ export default {
   created() {
     this.script = this.initial;
     document.addEventListener('keydown', this.switchPanel);
+    if (options.get('editorWindow') && global.history.length === 1) {
+      global.chrome.windows?.getCurrent({ populate: true }, setupSavePosition);
```

Comment:
How about using the `browser` API which is supposed to be the standard?

ps. maybe we should also handle the case where `chrome` does not exist on `global`, it's exposed in Firefox now but the document says it's not guaranteed in all browsers.

Reply:
It was a micro-optimization to avoid adding getCurrent to browser.js.